### DISCUSSION TITLE
New voice per chapter instead of new line

### DISCUSSION THREAD

**Discussion by the-lilith-cluster** [DrewThomasson/VoxNovel#12] [General]
Is it possible to have the split not happen per dialogue line but per chapter? I have a book that is first person POV and the character's name is in the chapter title. Having each of those be a unique voice would work really well for it.

**Comment by DrewThomasson** [comment]
Hmm I'll look into adding this seems like it could easily be done.

If this feature would be added how would you want the option in the gui?

I'm thinking about adding an option in the first book process window as a checkbox that says something like "Make each individual chapter have its own voice actor"

Thoughts?
